test/unit/gateways/checkout_v2_test.rb

```diff
@@ -1115,6 +1155,89 @@ def test_authorize_supports_alternate_credit_card_implementation
     end.respond_with(successful_authorize_response)
   end
 
+  def test_authorize_with_lvl_2_3_data
```

Comment:
For small words like level personally I don't like using abbreviation and also it would be good idea to follow the level2 and level3 standards in this kind if test.